src/contracts/tokens/ProtocolToken.sol

```diff
@@ -38,9 +45,21 @@ contract ProtocolToken is ERC20, ERC20Permit, ERC20Votes, Authorizable, IProtoco
     _burn(msg.sender, _wad);
   }
 
+  /// @inheritdoc IProtocolToken
+  function pause() external isAuthorized {
+    if (notPausable) revert ProtocolToken_NotPausable();
+    notPausable = true;
+    _pause();
+  }
+
+  /// @inheritdoc IProtocolToken
+  function unpause() external isAuthorized {
+    _unpause();
+  }
+
   // --- Overrides ---
 
-  function _update(address _from, address _to, uint256 _value) internal override(ERC20, ERC20Votes) {
+  function _update(address _from, address _to, uint256 _value) internal override(ERC20, ERC20Votes, ERC20Pausable) {
```

Comment:
q: what's `_update` in Pausable contract?

Reply:
`_update()` is overridden in `ERC20Pausable` so it includes the `whenNotPaused` modifier.

Reply:
ALSO MINT?!